ReactAndroid/build.gradle

```diff
@@ -16,30 +16,36 @@ import org.apache.tools.ant.filters.ReplaceTokens
 def downloadsDir = new File("$buildDir/downloads")
 def thirdPartyNdkDir = new File("$buildDir/third-party-ndk")
 
+// You need to have following folders in this directory:
+//   - boost_1_57_0
+//   - double-conversion-1.1.1
+//   - folly-deprecate-dynamic-initializer
+//   - glog-0.3.3
+//   - jsc-headers
+def dependenciesPath = System.getenv("REACT_NATIVE_DEPENDENCIES")
+
 // The Boost library is a very large download (>100MB).
 // If Boost is already present on your system, define the REACT_NATIVE_BOOST_PATH env variable
 // and the build will use that.
-def boostPath = System.getenv("REACT_NATIVE_BOOST_PATH")
+def boostPath = dependenciesPath ?: System.getenv("REACT_NATIVE_BOOST_PATH")
 
 task createNativeDepsDirectories {
     downloadsDir.mkdirs()
     thirdPartyNdkDir.mkdirs()
 }
 
 task downloadBoost(dependsOn: createNativeDepsDirectories, type: Download) {
-    // Use ZIP version as it's faster this way to selectively extract some parts of the archive
-    src 'https://downloads.sourceforge.net/project/boost/boost/1.57.0/boost_1_57_0.zip'
-    // alternative
-    // src 'http://mirror.nienbo.com/boost/boost_1_57_0.zip'
+    src 'https://github.com/react-native-community/boost-for-react-native/releases/download/v1.57.0-1/boost_1_57_0.tar.gz'
     onlyIfNewer true
     overwrite false
-    dest new File(downloadsDir, 'boost_1_57_0.zip')
+    dest new File(downloadsDir, 'boost_1_57_0.tar.gz')
```

Comment:
strange that we have to change this. Did sourceforge just remove this since we last published?